pandas/core/frame.py

```diff
@@ -8065,6 +8072,15 @@ def _join_compat(
             other = DataFrame({other.name: other})
 
         if isinstance(other, DataFrame):
+            if how == "cross":
```

Comment:
can you add an example inthe Examples section.

Reply:
Yep, done